### PR TITLE
Fixing outbox behavior so it displays correctly when inbox is empty

### LINKED ISSUE #196: Outbox only opens if there are messages in the Inbox
From an ask:
> ive noticed that outbox will only open if theres something in the inbox. its not a huge problem, but its a little inconvenient! (also "export configuration" is misspelled in the other tab but thats no big deal lmao)

### PR DESCRIPTION
**Pull request by blackjackkent**
Fixes #196

### CODE REVIEW

**Comment by bvtsang** [comment]
Don't forget to bump the version.

**Comment by blackjackkent** [comment]
Buggery. >.< Updated.

**Comment by bvtsang** [comment]
1. Empty your inbox
2. Visit your inbox (or refresh if you were already on the page)
3. See "No messages to display" message
4. Click My Outbox to view your outbox
5. Click My Outbox again to view your inbox
6. Expect to see "No messages to display" message
7. Observe that the "No messages to display" message is not there

**Comment by blackjackkent** [comment]
Good catch. Updated so that it's adding a div in addition to that instead of replacing the content in it.

File: Extensions/outbox.js [hunk 1]
```diff
@@ -475,7 +475,10 @@ XKit.extensions.outbox = new Object({
 			m_html = m_html + XKit.extensions.outbox.render(m_obj, m_message);
 
 		}
-
+		if ($('.no_posts_found').length > 0) {
+			$('.no_posts_found').after("<div id='posts' class='posts'></div>");
```
Comment (bvtsang) [inline Extensions/outbox.js]: I believe the DOM element that wraps the outbox elements is actually `<ol id="posts" class="posts post-avatar--static">`. I don't know if it matters, but the current implementation wraps the children `li` elements with a `div`, which seems improper.

**Comment by bvtsang** [comment]
LGTM

@homu r+

**Pull request merged**